src/main/java/org/embulk/util/aws/credentials/AwsCredentialsTask.java

```diff
@@ -37,4 +37,34 @@ public interface AwsCredentialsTask extends AwsCredentialsConfig {
     @Config("profile_name")
     @ConfigDefault("null")
     Optional<String> getProfileName();
+
+    @Override
+    @Config("aws_account_id")
+    @ConfigDefault("null")
+    Optional<String> getAwsAccountId();
+
+    @Override
+    @Config("role_name")
+    @ConfigDefault("null")
+    Optional<String> getRoleName();
+
+    @Override
+    @Config("external_id")
+    @ConfigDefault("null")
+    Optional<String> getExternalId();
+
+    @Override
+    @Config("duration_in_seconds")
+    @ConfigDefault("3600")
+    int getDurationInSeconds();
+
+    @Override
+    @Config("aws_partition")
+    @ConfigDefault("\"aws\"")
+    String getAwsPartition();
+
+    @Override
+    @Config("session_name")
+    @ConfigDefault("\"treasure-data\"")
```

Comment:
Having `treasure-data` here does not sound good to me. This project is OSS Embulk's not Treasure Data's.

Reply:
@minidragon88

Reply:
So could you propose another name, is `embulk` should reasonable? @dmikurube

Reply:
I have changed default value to `embulk`